apptools/feedback/feedbackbot/tests/test_model.py

```diff
@@ -0,0 +1,43 @@
+"""
+Tests for FeedbackMessage model.
+"""
+
+import numpy as np
+import unittest
+
+from traits.api import TraitError
+from traits.testing.unittest_tools import UnittestTools
+
+from apptools.feedback.feedbackbot.model import FeedbackMessage
+
+
+class TestFeedbackMessage(unittest.TestCase, UnittestTools):
```

Comment:
These are essentially testing traits. It would be better to test the logic in the `send` method. This can be done by mocking `slack.WebClient.files_upload` and checking that the right information is passed.